src/test/java/com/werfen/report/CompareExcelFiles.java

```diff
@@ -0,0 +1,105 @@
+
+        int sheetsInWorkbook1 = workbook1.getNumberOfSheets();
+        int sheetsInWorkbook2 = workbook2.getNumberOfSheets();
+        assertEquals(sheetsInWorkbook1, sheetsInWorkbook2,
+                "Excel work books have different number of sheets. \n "
+                        + "Sheets in work book 1 : " + sheetsInWorkbook1 + "\n "
+                        + "Number of sheets in work book 2 : " + sheetsInWorkbook2);
+        List<String> sheetsNameOfWb1 = new ArrayList<>();
+        List<String> sheetsNameOfWb2 = new ArrayList<>();
+
+        for (int i = 0; i < sheetsInWorkbook1; i++) {
+            sheetsNameOfWb1.add(workbook1.getSheetName(i));
+            sheetsNameOfWb2.add(workbook2.getSheetName(i));
+        }
+
+        Collections.sort(sheetsNameOfWb1);
+        Collections.sort(sheetsNameOfWb2);
+        assertEquals(sheetsNameOfWb1, sheetsNameOfWb2, "Provided excel work books have different name of sheets.");
+    }
+
+    public void verifySheetsInExcelFilesHaveSameRowsAndColumns(Workbook workbook1, Workbook workbook2) {
+        int sheetCounts = workbook1.getNumberOfSheets();
+
+        for (int i = 0; i < sheetCounts; i++) {
+            Sheet s1 = workbook1.getSheetAt(i);
+            Sheet s2 = workbook2.getSheetAt(i);
+            int rowsInSheet1 = s1.getPhysicalNumberOfRows();
+            int rowsInSheet2 = s2.getPhysicalNumberOfRows();
+            assertEquals(rowsInSheet1, rowsInSheet2, "Sheets have different count of rows..");
+
+            Iterator<Row> rowInSheet1 = s1.rowIterator();
+            Iterator<Row> rowInSheet2 = s2.rowIterator();
+            while (rowInSheet1.hasNext()) {
+                int cellCounts1 = rowInSheet1.next().getPhysicalNumberOfCells();
+                int cellCounts2 = rowInSheet2.next().getPhysicalNumberOfCells();
+                assertEquals(cellCounts1, cellCounts2, "Sheets have different count of columns..");
+            }
+        }
+    }
+
+    public void verifyDataInExcelBookAllSheets(Workbook workbook1, Workbook workbook2) {
+        int sheetCounts = workbook1.getNumberOfSheets();
+
+        for (int i = 0; i < sheetCounts; i++) {
+            Sheet s1 = workbook1.getSheetAt(i);
+            Sheet s2 = workbook2.getSheetAt(i);
+            int rowCounts = s1.getPhysicalNumberOfRows();
+
+            for (int j = 0; j < rowCounts; j++) {
+                if (nonNull(s1.getRow(j)) && nonNull(s2.getRow(j))) {
+                    int cellCounts = s1.getRow(j).getPhysicalNumberOfCells();
+                    for (int k = 0; k < cellCounts; k++) {
+                        Cell c1 = s1.getRow(j).getCell(k);
+                        Cell c2 = s2.getRow(j).getCell(k);
+                        if (nonNull(c1) && nonNull(c2)) {
+                            if (c1.getCellType().equals(c2.getCellType())) {
+                                if (c1.getCellType() == CellType.STRING) {
+                                    String v1 = c1.getStringCellValue();
```

Comment:
I would suggest centralizing the obtention of a cell value (as a string) in a private method. For instance, we could have something like:
```java
private String getCellValue(Cell c) {
    switch (c.cellType()) {
        case  CellType.NUMERIC:
            return c.getStringCellValue();
        case CellType.STRING:
            return c.getNumericCellValue().toString();
        ...
     } 
   }
```